webapp/src/dogma/common/components/ErrorWrapper.tsx

```diff
@@ -0,0 +1,20 @@
+import { ReactNode, useEffect } from 'react';
+import { useToast } from '@chakra-ui/react';
+import { useAppSelector } from 'dogma/store';
+
+export const ErrorWrapper = (props: { children: ReactNode }) => {
+  const { errorText, errorType } = useAppSelector((state) => state.message);
+  const toast = useToast();
+  useEffect(() => {
+    if (errorText) {
+      toast({
+        title: errorType,
```

Comment:
Status icons imply that an error type is warn or info.
How about exposing `title` only if a meaningful value is set?

Reply:
Updated message slice to allow the caller to provide title, text and type.